templates/UmbracoExtension/Client/openapi-ts.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from '@hey-api/openapi-ts';
+
+export default defineConfig({
+    client: '@hey-api/client-fetch',
+    input: 'https://localhost:44339/umbraco/swagger/example/swagger.json',
```

Comment:
This is the port number of your test website - it will be different for users of the template. I guess we would need to add a new parameter so that they can specify the website's development url when running the `dotnet new umbraco-extension` command. Can't think of any other way around this!

Reply:
As agreed on our call Lotte we will use a flag/switch to set the domain prefix `https://localhost:44339`